### DISCUSSION TITLE
High CPU in self hosted mode.

### DISCUSSION THREAD

**Discussion by ghvader** [gtxaspec/wz_mini_hacks#717] [General]
I noticed wihout self hosted mode on, cpu for icamera is about 40-50% but if you enable self hosted mode, it doubles and runs between 85 and 90% anyone else see this? is it expected?

**Comment by amandamana** [comment]
I see about 25-35% cpu usage on my SELFHOSTED_MODE v3s serving single RTSP_HI_RES streams with no audio. Perhaps a bit lower than typical as I run fps=10 and presently with the webserver off. MASwitch, AASwitch, SASwitch, and CASwitch all disabled, and an NTP server in my LAN, if that matters... Also note these have never touched the cloud, p2pid= and r_encr= are blank.

**Comment by digitalpixies** [comment]
my wyze v3 runs at 100% at the beginning. I checked 20mins later (uptime) and it's dropped down to around 50%. Does yours remain at 100% past the 20min mark?
